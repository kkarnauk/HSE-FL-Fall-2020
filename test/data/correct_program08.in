a (a) :- a.
a (a (a) (a a)) :- ((((a)))).
a :- aaa (aaa (aaa)) (aaaaa).
