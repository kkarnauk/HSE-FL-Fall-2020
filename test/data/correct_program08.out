|relation
|	|head
|	|	|atom
|	|	|	|identifier = 'a'
|	|	|	|atom
|	|	|	|	|identifier = 'a'
|	|body
|	|	|identifier = 'a'

|relation
|	|head
|	|	|atom
|	|	|	|identifier = 'a'
|	|	|	|atom
|	|	|	|	|identifier = 'a'
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'a'
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'a'
|	|	|	|	|	|identifier = 'a'
|	|body
|	|	|identifier = 'a'

|relation
|	|head
|	|	|identifier = 'a'
|	|body
|	|	|atom
|	|	|	|identifier = 'aaa'
|	|	|	|atom
|	|	|	|	|identifier = 'aaa'
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'aaa'
|	|	|	|atom
|	|	|	|	|identifier = 'aaaaa'